docs/utils/document.js

```diff
@@ -1,6 +1,6 @@
 const path = require("path");
 
-const rePathNumbers = new RegExp(`(${path.sep})\\d+-`, "g");
+const rePathNumbers = new RegExp(`(^|${path.sep})\\d+-`, "g");
```

Comment:
This was needed so that `omitNumbers` works for `01-*` as well, although I'm not sure if it's needed now, but it doesn't hurt.

Reply:
I don't see any cases where omitNumbers is passed a relative path where it might be an issue, but I don't suppose it hurts.

While looking it up, i did notice we now have two `omitNumbers` functions. I'd get rid of the one in `docs/src/utils` since it doesn't seem to be used anywhere. Nor is `capitalize` in that file, though it doesn't have another function it might get confused with.

Reply:
In the test environment we use relative paths, see in [`gatsby-node.js`](https://github.com/kiwicom/orbit/pull/3005/files#diff-97ce157ffa078c33fbccfb2e69ae8be278c4074fbd0ec7c750ce35cceaa454f1R193).